From 1d00a5bf05f46e51cccf565e47a156f126805c3d Mon Sep 17 00:00:00 2001
From: Glen Kuhne <kuh@google.com>
Date: Mon, 2 Oct 2017 09:46:59 -0700
Subject: [PATCH 07/10] klte-common: Add p2p_no_group_iface=1 to
 p2p_supplicant_overlay

Configures wpa_supplicant to use p2p0 mgmt iface instead of dynamically
created p2p-p2p0-XYZ virtual interfaces for P2P group.

Bug: 65524423
Test: CtsVerifier WifiDirect
Test: Manual verification
Change-Id: I70582f114ab69684424aac41fbeb78fa7cc04847
Merged-In: Id9add59aa122adb7f39c80252061a8f5d3cc55cd
---
 configs/p2p_supplicant_overlay.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/p2p_supplicant_overlay.conf b/configs/p2p_supplicant_overlay.conf
index 37f7a5b..71f2e4f 100644
--- a/configs/p2p_supplicant_overlay.conf
+++ b/configs/p2p_supplicant_overlay.conf
@@ -1,2 +1,3 @@
 disable_scan_offload=1
 p2p_search_delay=100
+p2p_no_group_iface=1
-- 
2.17.1

